src/main/java/com/autotune/analyzer/services/ListStackLayers.java

```diff
@@ -122,12 +145,16 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws IO
     }
 
     private void addAppLayersToResponse(JSONArray outputJsonArray, String autotuneObjectKey, AutotuneObject autotuneObject, String layerName) {
-        JSONObject jsonObject = new JSONObject();
-        addExperimentDetails(jsonObject, autotuneObject);
+        JSONObject experimentJson = new JSONObject();
+        addExperimentDetails(experimentJson, autotuneObject);
+
+        JSONArray stackArray = new JSONArray();
 
-        JSONArray layersArray = new JSONArray();
-        for (String applicationServiceStackName : applicationServiceStackMap.get(autotuneObjectKey).keySet()) {
-            ApplicationServiceStack applicationServiceStack = applicationServiceStackMap.get(autotuneObjectKey).get(applicationServiceStackName);
+        for (String containerImageName : applicationServiceStackMap.get(autotuneObjectKey).keySet()) {
```

Comment:
Add null checks to hash maps

Reply:
In all APIs as applicable

Reply:
Have already added in the last commit, please refresh